### DISCUSSION TITLE
minemacs configuration seems to interpret words as camelcase subsets

### DISCUSSION THREAD

**Discussion by eirik-r** [abougouffa/minemacs#152] [Q&A]
E.g. yw on ITabBar will return 'I', same with searches etc. How can this be modified back to "normal" mode?

**Comment by abougouffa** [comment]
Hello @eirik-r 

I think this is not a MinEmacs issue, it seems to be the default behavior of [Evil](https://github.com/emacs-evil/evil). I don't know how to fix this but I think this answer can be helpful https://emacs.stackexchange.com/questions/14439/evil-package-alternative-for-motion-through-camelcase-words.
